### DISCUSSION TITLE
Updating the chia node while plotting?

### DISCUSSION THREAD

**Discussion by TheRealGioviok** [swar/Swar-Chia-Plot-Manager#204] [Q&A]
Hi.
I was wandering if it could be possible to update the chia node while plotting with the manager, or at least to update without interrupting plots that already began plotting.

**Comment by harley19801** [comment]
No you can't do that, because chia plot exe file will be updated.

**Reply by harley19801** [reply]
Here is a such case:
https://github.com/swar/Swar-Chia-Plot-Manager/discussions/189
